cfgov/unprocessed/apps/owning-a-home/js/explore-rates/index.js

```diff
@@ -531,27 +465,27 @@ function checkForJumbo() {
   var request;
 
   loan = jumbo( {
-    loanType:   params['loan-type'],
-    loanAmount: params['loan-amount']
+    loanType:   params.getVal( 'loan-type' ),
+    loanAmount: params.getVal( 'loan-amount' )
   } );
   dropdown( 'loan-type' ).enable( norms );
   dropdown( 'loan-type' ).hideHighlight();
   $( '#county-warning' ).addClass( 'u-hidden' );
 
   // if loan is not a jumbo, hide the loan type warning
-  if ( jQuery.inArray( params['loan-type'], jumbos ) < 0 ) {
+  if ( jQuery.inArray( params.getVal( 'loan-type' ), jumbos ) < 0 ) {
```

Comment:
Any idea why it's `jQuery` here and `$` everywhere else?

Reply:
Hm, not sure. That's the original setup. It seems like it's just aliased to `$` https://github.com/cfpb/owning-a-home/blob/master/src/static/js/modules/explore-rates.js#L4

Reply:
There's even another use of `$.inArray()` farther down the code base. Mind updating it to conform with the rest?

Reply:
Removed `jQuery` variable in https://github.com/cfpb/cfgov-refresh/pull/3830/commits/69528e828534f09ee491cc38775bb3697a136043